bindings/python/tests/bindings/test_tokenizer.py

```diff
@@ -152,8 +152,6 @@ def test_encode(self):
         assert len(output) == 2
 
     def test_encode_formats(self, bert_files):
```

Comment:
@ArthurZucker there are already a set of tests covering `np.array` - did you have something else in mind?

Reply:
Nope I checked and found then so approved and good to go!